config/checkstyle_checks.xml

```diff
@@ -740,6 +740,12 @@
       <property name="format" value="[^\p{ASCII}]"/>
       <property name="ignoreComments" value="true"/>
     </module>
+    <module name="RegexpSinglelineJava">
+      <property name="id" value="noUsageOfGetFileContents()Method"/>
+      <property name="format" value="getFileContents()"/>
```

Comment:
Since this is regexp, don't we have to sanitize the `()` since we want to match text?